### DISCUSSION TITLE
The process has been signaled with signal "6".

### DISCUSSION THREAD

**Discussion by henrikruscon** [spatie/browsershot#707] [Bugs]
Just trying out `Browsershot::html('<h1>Hello world!!</h1>')->save('example.pdf');` returns `The process has been signaled with signal "6".` error.

What could it be?

**Comment by henrikruscon** [comment]
Tried outputting the process to see what it the cmd:

```php
$process = Process::fromShellCommandline($fullCommand)->setTimeout($this->timeout);

dd($process);
```

I tried running the output directly in my terminal and it worked without any issues at all. What could be wrong here? Something with puppeteer but only from inside vendor somehow? Since it works from the root of the project from terminal directly.

Would really appreciate help here.

**Comment by henrikruscon** [comment]
From debugging some more I discovered that `Process::fromShellCommandline` with no command even throws a signal 6. Now I really have no idea what it is.

**Comment by henrikruscon** [comment]
This was the issue #580

**Comment by PatrickSJ** [comment]
What OS is this running, what version of the OS, and which versions of nodejs & npm?  I have seen this error occur when the nodejs/npm binaries are not compatible with specific versions of the OS.
